test.js

```diff
@@ -2,6 +2,33 @@ var assert = require( "assert" );
 var zlib = require( "zlib" );
 var s = require( "./index" );
 var type;
+let expectedCSVData = [
+    {
+        Year: '1997',
+        Make: 'Ford',
+        Model: 'E350',
+        Description: 'ac, abs, moon',
+        Price: '3000.00'
+    }, {
```

Comment:
Again, for being consistent with the coding style in this repo I would suggest following the same bracket/object/line-break that exists below (https://github.com/avinoamr/structurize/blob/master/test.js#L61-L100)